styles/style.css

```diff
@@ -329,8 +390,6 @@ animation: rotation 20s linear infinite;
   transition: opacity .3s ease;
```

Comment:
And here,
Right now, the smooth transition is only applied when you hover over the link. When you move the cursor away, the transition becomes abrupt. To make it work smoothly in both directions, move the `transition` property from the `:hover` state to the element itself.